modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/spi/model/result/contributor/CPDefinitionModelSummaryContributor.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.product.internal.search.spi.model.result.contributor;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.search.Document;
+import com.liferay.portal.kernel.search.Field;
+import com.liferay.portal.kernel.search.Summary;
+import com.liferay.portal.search.spi.model.result.contributor.ModelSummaryContributor;
+
+import java.util.Locale;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Brian I. Kim
+ */
+@Component(
+	property = "indexer.class.name=com.liferay.commerce.product.model.CPDefinition",
+	service = ModelSummaryContributor.class
+)
+public class CPDefinitionModelSummaryContributor
+	implements ModelSummaryContributor {
+
+	@Override
+	public Summary getSummary(
+		Document document, Locale locale, String snippet) {
+
+		String prefix = Field.SNIPPET + StringPool.UNDERLINE;
+
+		String title = document.get(prefix + Field.NAME, Field.NAME);
+		String content = document.get(
```

Comment:
please make sure this is also fixing this https://liferay.atlassian.net/browse/COMMERCE-11771

Reply:
I will fix as part of this ticket, but the solution for that ticket is not tied to search @andrea-ale-sbarra .